certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3443 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.4",
+    "title": "Virus Total APIV3",
```

Comment:
Any reason to club `API` and `V3` into a single word? Can the title be changed to `VirusTotal API version 3` as in https://developers.virustotal.com/v3.0/reference#overview?

Reply:
Done